templates/definition/meter/deye-mi.yaml

```
@@ -0,0 +1,43 @@
+template: deye-mi
```

```suggestion
template: deye-micro
```

Wir müssen nicht ganz so kryptisch sein?

---

Die heissen bei Deye in der Doku so?

---

Ah, das wusste ich nicht- hatte aufs micro geschaut.